src/docfx/lib/github/UserProfileCache.cs

```diff
@@ -16,11 +16,13 @@ internal class UserProfileCache
         private readonly ConcurrentDictionary<string, UserProfile> _cacheByName;
         private readonly ConcurrentDictionary<string, UserProfile> _cacheByEmail;
         private readonly string _cachePath;
+        private readonly GitHubAccessor _github;
 
-        public UserProfileCache(IDictionary<string, UserProfile> cache, string path)
+        public UserProfileCache(IDictionary<string, UserProfile> cache, string path, GitHubAccessor github)
```

Comment:
>public [](start = 8, length = 6)

should this be private?

Reply:
yes~

---
In reply to: [208164576](https://github.com/dotnet/docfx/pull/3163#discussion_r208164576) [](ancestors = 208164576)